src/goLanguageServer.ts

```diff
@@ -311,39 +298,42 @@ export function getLanguageServerToolPath(): string {
 		return;
 	}
 
-	// Get the path to gopls or any alternative that the user might have set for gopls.
-	const goplsBinaryPath = getBinPath('gopls');
-	if (path.isAbsolute(goplsBinaryPath)) {
-		return goplsBinaryPath;
-	}
```

Comment:
There would still be merit in having the above. With the changes in this PR, if `goConfig['alternateTools']` is null or undefined, then we end up prompting the user to install `gopls` even if it was already installed

cc @stamblerre, @hyangah